Modules/Scripted/DICOMLib/DICOMWidgets.py

```diff
@@ -491,18 +491,22 @@ def promptForDatabaseDirectory(self):
         But, if the application is in testing mode, just pick a temp directory.
     """
     if slicer.app.commandOptions().testingEnabled:
-      databaseDirectory = os.path.join(slicer.app.temporaryPath, 'tempDICOMDatbase')
+      databaseDirectory = os.path.join(slicer.app.temporaryPath, 'tempDICOMDatabase')
     else:
       databaseDirectory = self.settings.value('DatabaseDirectory')
-      if not databaseDirectory:
-        documentsLocation = qt.QDesktopServices.DocumentsLocation
-        documents = qt.QDesktopServices.storageLocation(documentsLocation)
-        databaseDirectory = os.path.join(documents, "SlicerDICOMDatabase")
-        message = "DICOM Database will be stored in\n\n{}\n\nUse the Local Database button in " \
-                  "the DICOM Browser to pick a different location.".format(databaseDirectory)
-        slicer.util.infoDisplay(message, windowTitle='DICOM')
+      documentsLocation = qt.QDesktopServices.DocumentsLocation
```

Comment:
I did not move initialization of the variables, they remain in the same places where they were in the original version. It looks like I moved them, but I just changed the spacing. I think the location is alright, not sure what can be improved here.